docs/source/CONTRIBUTING.md

```diff
@@ -1 +1 @@
-../../CONTRIBTING.md
\ No newline at end of file
+../../CONTRIBUTING.md
```

Comment:
I saw 2 warnings treated as errors in the docs builds:

```text
/__w/legate-raft/legate-raft/docs/source/CONTRIBUTING.md:25: WARNING: Unknown source document '/__w/legate-raft/legate-raft/README' [myst.xref_missing]
/__w/legate-raft/legate-raft/docs/source/README.md:49: WARNING: Unknown source document '/__w/legate-raft/legate-raft/CONTRIBTING' [myst.xref_missing]
```

([build link](https://github.com/rapidsai/legate-raft/actions/runs/13462287106/job/37622566430?pr=14#step:6:2969))

Renaming `CONTRIBTING.md` to `CONTRIBUTING.md` should fix one of them (the reference in `docs/source/index.rst` was inconsistent with the actual file name), and maybe both?

Reply:
Ahhhh ok, so I still think this renaming (to fix the spelling error) was a good thing, but it did not resolve the failure.

Interesting to note that the docs this is complaining about are totally working in the rendered docs. You can go to https://rapidsai.github.io/legate-raft/README.html and try clicking "our guide for contributing to legate-raft" ... it leads to https://rapidsai.github.io/legate-raft/CONTRIBUTING.html successfully.

Reply:
Oh! @seberg I just saw your comment here: https://github.com/rapidsai/legate-boost/pull/212#issuecomment-2678694098

I guess it's `sphinx` 8.2.0 that is causing these warnings? I'll try pinning here too.

Reply:
Right, that was one of the things I saw in legate-df.

Reply:
Ah got it, ok! I've pushed a `sphinx<8.2.0` pin here, it looks like it'd fix docs builds: https://github.com/rapidsai/legate-raft/actions/runs/13502337421/job/37726967372?pr=14